lnrpc/rpc.proto

```diff
@@ -798,6 +800,7 @@ message NewWitnessAddressRequest {
 message NewAddressResponse {
     /// The newly generated wallet address
     string address = 1 [json_name = "address"];
+    string chain = 2 [json_name = "chain"];
```

Comment:
The request will have an optional ticker option to select the chain. This is not part of this PR (will come later). Regarding the response - IMHO it is better to be included in the response so that the user will have more understanding. While it can easily be removed I think we should leave it there.

Reply:
Note that the ticker option of the request is optional. This is another reason why the chain should be included in the response:

lncli newaddress np2wkh
{
    "address": "2MtuEcJMMvZJ9LibSXJ2gSSNcXfqwi7SRiL",
    "chain": "bitcoin"
}